Test/datatypes/t4.bpl

```diff
@@ -0,0 +1,42 @@
+// RUN: %parallel-boogie "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+type{:datatype} Pair;
```

Comment:
What's the benefit of `type{:datatype} Pair` over `datatype Pair` ?

Reply:
The syntax for specifying a datatype is not introduced by this PR. It has existed for a while. The original introduction of datatypes in Boogie was experimental with no investment in syntax. So everything was done by using attributes on existing syntax. What you are seeing is a result of that design. It is desirable to have a nice clean syntax for specifying datatypes. Perhaps the subject of another PR.